manifests/backup/backup-s3.yaml

```diff
@@ -5,12 +5,16 @@ metadata:
   name: demo1-backup-s3
   namespace: test1
 spec:
+  from:
+    host: 10.0.0.1
+    port: 4000
+    user: root
+    secretName: backup-demo1-tidb-secret
```

Comment:
use `passwordSecretName`?

Reply:
I don't think it makes sense to include the name of the key in the `secretName`. What if the secret stores more stuff?